packages/phone-number-privacy/combiner/src/config.ts

```diff
@@ -1,3 +1,4 @@
+import { toBool } from '@celo/phone-number-privacy-common/lib/utils/config-utils'
```

Comment:
Can you set have the common's `index.ts` export things so we don't need to reach into `lib/...` like this. 
Here's an example:
https://github.com/celo-org/celo-monorepo/blob/master/packages/utils/src/index.ts#L1

Reply:
Thanks! Missed the export for this file